palette/src/rgb/mod.rs

```diff
@@ -2,30 +2,30 @@
 
 use core::any::Any;
 
-use white_point::WhitePoint;
-use {Component, FloatComponent, FromComponent, Yxy};
-
-use encoding::{Linear, TransferFn};
+use crate::encoding;
+use crate::encoding::{Gamma, Linear, TransferFn};
```

Comment:
Just FYI, this is possible to write as just `use crate::encoding::{self, Gamma, Linear, TransferFn};` if one would want to have fewer lines. :slightly_smiling_face: It's perfectly fine like this too.